src/views/virtualmachinesinstance/details/tabs/disks/utils/virtualMachinesInstancePageDisksTabUtils.ts

```diff
@@ -0,0 +1,38 @@
+import byteSize from 'byte-size';
+
+import { V1Disk } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+export type DiskPresentation = {
+  name: string;
+  interface: string;
+  drive: string;
+  source: string;
+  storageClass?: string;
+  size?: string;
+  namespace?: string;
+};
+
+const isDiskOrCdrom = (obj: object) => (obj.hasOwnProperty('disk') ? 'disk' : 'cdrom');
```

Comment:
please change `object` to `V1Disk`
I would also flip the condition, more chances its a disk rather than a cdrom.
`const isDiskOrCdrom = (obj: V1Disk) => (obj?.cdrom ? 'cdrom' : 'disk');`

Reply:
actually i think i will Proxy it as i just saw we have also floppy and LUN